galaxy_importer/main.py

```diff
@@ -98,27 +107,43 @@ def call_importer(args, cfg):  # pragma: no cover
 
     Method excluded from pytest unit test coverage, tests exist in tests/integration
     """
-    if not args.file:
-        return collection.import_collection(
-            git_clone_path=os.path.abspath(args.git_clone_path),
-            output_path=os.path.abspath(args.output_path),
-            logger=logger,
-            cfg=cfg,
-        )
-
-    match = FILENAME_REGEXP.match(os.path.basename(args.file))
-    namespace, name, version = match.groups()
-    filename = collection.CollectionFilename(namespace, name, version)
-
-    with open(args.file, "rb") as fh:
+    if args.legacy_role:
+        if args.file is None:
+            logger.error("Must supply the directory of the role")
```

Comment:
these two commands determin the role name as expected
```
(.env) awc:~$ python -m galaxy_importer.main --legacy-role code/ansible-role-jenkins --namespace awcrosby
(.env) awc:code$ python -m galaxy_importer.main --legacy-role ansible-role-jenkins --namespace awcrosby
```
but this does not
```
(.env) awc:ansible-role-jenkins$ python -m galaxy_importer.main --legacy-role . --namespace awcrosby
Importing with galaxy-importer 0.4.10
Determined role name to be .
```

Reply:
we don't need to support running this from any directory depth, especially since this is a legacy feature. my recommendation:

1. in the readme and in the `import_legacy_role` docstring, say you need to run this inside the parent directory of the role
2. have the importer fail if the directory depth is not equal to 1... that way users are not missing an error about their role name (since the importer thought it was `.`) or they are not missing lint output